### PR TITLE
DO NOT MERGE

### PR DESCRIPTION
**Pull request by armantajback**

### CODE REVIEW

File: codalab/lib/bundle_cli.py [hunk 1]
```diff
@@ -1823,6 +1824,53 @@ def do_rm_command(self, args):
...
+            curr_bundle_uuid, level = bundle_stack.pop()
+            # Assume only 1 bundle in bundle array
+            curr_bundle = client.fetch('bundles', params={'specs': curr_bundle_uuid})[0]
+            if curr_bundle_uuid in processed_bundles:
+                print('{}- {}({})...'.format(
+                    " "*level,
+                    curr_bundle['metadata']['name'],
+                    worksheet_util.apply_func(UUID_POST_FUNC, curr_bundle_uuid)), file=self.stdout)
+            else:
+                print('{}- {}({})'.format(
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Try to avoid code duplication here.

Comment (armantajback) [reply codalab/lib/bundle_cli.py]: I've refactored it to use a private to the method function

File: codalab/lib/bundle_cli.py [hunk 2]
```diff
@@ -1823,6 +1824,53 @@ def do_rm_command(self, args):
...
+        bundle_uuids = self.target_specs_to_bundle_uuids(client, worksheet_uuid, args.bundle_spec)
+        bundle_stack.extend([(uuid, 0) for uuid in bundle_uuids])
+
+        # Traversal through all ancestors, avoid printing entire tree on repeated worksheets
+        while (len(bundle_stack) != 0):
+            curr_bundle_uuid, level = bundle_stack.pop()
+            # Assume only 1 bundle in bundle array
+            curr_bundle = client.fetch('bundles', params={'specs': curr_bundle_uuid})[0]
+            if curr_bundle_uuid in processed_bundles:
+                print_ancestor('{}- {}({})...', level, curr_bundle['metadata']['name'], curr_bundle_uuid)
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: I'd do this: `print(render(curr_bundle) + ('...' if curr_bundle_uuid in processed_bundles else ''))`

Comment (armantajback) [reply codalab/lib/bundle_cli.py]: Is the render method you mention equivalent to the print_ancestor function (should it take the full list of params)?

Comment (percyliang) [reply codalab/lib/bundle_cli.py]: The main thing I'm trying to get at is trying to have more code sharing (which means minimizing the arguments - I guess you need `level`).

File: docs/CLI-Reference.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ This file is auto-generated from the output of `cl help -v` and provides the lis
           -p, --pack               If path is an archive file (e.g., zip, tar.gz), keep it packed.
           -z, --force-compression  Always use compression (this may speed up single-file uploads over a slow network).
           -w, --worksheet-spec     Upload to this worksheet ([(<alias>|<address>)::](<uuid>|<name>)).
+          -i, --ignore             Name of file containing patterns matching files and directories to exclude from upload. This option is currently only supported with the GNU tar library.
```
Comment (armantajback) [inline docs/CLI-Reference.md:24]: @percyliang  Do you know why these files were generated when I ran the precommit script? I don't see anything in the diff that explains them, perhaps I can try a rebase.

Comment (yipenghe) [reply docs/CLI-Reference.md:24]: precommit.sh auto generates some docs, the current master branch should have this change already so rebasing should do it

**Pull request closed without merging**